.github/actions/vs-shell/README.md

```diff
@@ -0,0 +1,39 @@
+Visual Studio shell
```

Comment:
I've added `egor-tensin/vs-shell` as an allowed source so I think it would be possible to remove this. I think that's a better option since maintaining a fork of this is very much outside the scope of the library itself.

Reply:
I think it makes sense to also specify the desired version of the 3rd party action. Though, apparently, your allow list is too restrictive for that:

> Bad request - egor-tensin/vs-shell@v2 is not allowed to be used in randombit/botan. Actions in this workflow must be: within a repository owned by randombit, created by GitHub, or matching the following: google/oss-fuzz/*, egor-tensin/vs-shell.

I guess you want to allow `egor-tensin/vs-shell*`, `.../vs-shell@*` or even `.../vs-shell@2`...

Reply:
@randombit I need your help with a repository config. :)

Reply:
Sorry I missed this! I updated the rule to `egor-tensin/vs-shell@*`

Reply:
The 3rd party usage seems to work now. But something still fails in the windows builds. I'll look into it tomorrow. Thanks for giving this another look with the configs.

Reply:
Heureka!